Index: components/com_acymailing/test1.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- components/com_acymailing/test1.php	(date 1474948145000)
+++ components/com_acymailing/test1.php	(date 1474948145000)
@@ -0,0 +1,1 @@
+
\ No newline at end of file
Index: components/com_acymailing/test2.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- components/com_acymailing/test2.php	(date 1474948169000)
+++ components/com_acymailing/test2.php	(date 1474948169000)
@@ -0,0 +1,1 @@
+
\ No newline at end of file
